CHANGELOG.md

```diff
@@ -27,10 +27,16 @@ The following changes have been implemented but not released yet:
 
 ### Breaking Changes
 
-- Parsing Verifiable Credentials. This allows the Verifiable Credential to be read using the RDF/JS DatasetCore API. This is a breaking change because the `VerifiableCredential` type now is also of type `DatasetCore`. Importantly, this dataset is not preserved when converting to verifiableCredentials a string and back doing `JSON.parse(JSON.stringify(verifiableCredential))`. We reccomend that developers set `returnLegacyJsonld` to `false` in functions such as `getVerifiableCredential` in order to avoid returning deprecated object properties. Instead developers should make use of the exported `getter` functions to get these attributes.
+- Parsing Verifiable Credentials. This allows the Verifiable Credential to be read using the RDF/JS DatasetCore API.
+  This is a breaking change because the `VerifiableCredential` type now is also of type `DatasetCore`. Importantly, this
+  dataset is not preserved when converting to verifiableCredentials a string and back doing
+  `JSON.parse(JSON.stringify(verifiableCredential))`. We reccomend that developers set `returnLegacyJsonld` to `false`
+  in functions such as `getVerifiableCredential` in order to avoid returning deprecated object properties. Instead
```

Comment:
It's funny these had never been caught by our linter. There probably is a misconfiguration. Am I correct to assume you IDE is configured to run prettier on save, regardless of the current linting config?

Reply:
Not quite. I think a new version of prettier (or just the config)  is objecting to warnings so I had to reformat it to get lint to pass. I also had to suppress the warning on a skipped test.